src/DynamoCoreWpf/Views/GuidedTour/PopupWindow.xaml.cs

```diff
@@ -91,6 +91,8 @@ private void PopupWindow_Closed(object sender, EventArgs e)
             if (webBrowserComponent != null)
             {
                 webBrowserComponent.Visibility = Visibility.Collapsed;
+                webBrowserComponent.Dispose();
+                GC.SuppressFinalize(webBrowserComponent);
```

Comment:
I did not dispose of this before because it is difficult to tell how the PopupWindow is used/re-used